public/api/conf/3.0/application.raml

```diff
@@ -448,8 +448,8 @@ documentation:
               body:
                 application/xml:
                   description: Successful response payload
-                  type: !include schemas/wco/status/StatusQueryResponse.xsd
-                  example: !include schemas/wco/status/StatusQueryResponseExample.xml
+                  type: !include schemas/wco/status/DeclarationStatusResponse.xsd
```

Comment:
Do you know why v2 has this xsd and example but no mention of it in v2 raml?

Reply:
I'm assuming you mean v3 has and v2 doesn't. Status is not enabled in v2 - but the examples and schemas are there to keep them all consistent.